From b34a51794b78a19cce57a4d241ee551defb7430e Mon Sep 17 00:00:00 2001
From: Wenping Zhang <wenping.zhang@rock-chips.com>
Date: Tue, 25 Jul 2017 17:55:06 +0800
Subject: [PATCH] ARM: dts: rk3229-gva-sdk: update changes for new gva
 hardware.

1. use emmc instead of nand.
2. enable cypress cy8c4024 touchpad support.

Change-Id: Ie9e72516601f2af8caaa1325b98d1574d010a22f
Signed-off-by: Wenping Zhang <wenping.zhang@rock-chips.com>
---
 arch/arm/boot/dts/rk3229-gva-sdk.dts | 15 +++++++++++++--
 1 file changed, 13 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/rk3229-gva-sdk.dts b/arch/arm/boot/dts/rk3229-gva-sdk.dts
index 34b5717df127..a70b617ef0ae 100644
--- a/arch/arm/boot/dts/rk3229-gva-sdk.dts
+++ b/arch/arm/boot/dts/rk3229-gva-sdk.dts
@@ -128,11 +128,11 @@
 };
 
 &nandc {
-	status = "okay";
+	status = "disabled";
 };
 
 &emmc {
-	status = "disabled";
+	status = "okay";
 };
 
 &i2c0 {
@@ -272,6 +272,17 @@
 	};
 };
 
+&i2c1 {
+	status = "okay";
+	ts@8 {
+		compatible = "cypress,cy8c4024";
+		reg = <0x8>;
+		interrupt-parent = <&gpio1>;
+		interrupts = <RK_PA1 IRQ_TYPE_LEVEL_LOW>;
+		status = "okay";
+	};
+};
+
 &i2c2 {
 	status = "okay";
 
-- 
2.35.3

